spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -384,6 +394,13 @@
       expect(subject).to receive(:ab_finished).with(:demo)
       get :new
     end
+
+    it 'tracks the pageview' do
```

Comment:
Good point, but I think we would still need a before_action for each controller, and only on the endpoints accessible via GET.